public/js/controllers/charts_controller.js

```diff
@@ -597,7 +635,7 @@ export default class extends Controller {
   }
 
   isTimeAxis () {
-    return this.selectedAxis() === 'time'
+    return this.selectedAxis() === 'time' && customXLabel(this.chartSelectTarget.value) !== ''
```

Comment:
This test `customXLabel(this.chartSelectTarget.value) !== ''` is used in several functions to dictate different behaviors.  If this test were made into a function, what would it be called?  `specialXAxis` perhaps, or something more descriptive (I don't know)?  My concern is that it's just `'duration-btw-blocks'` right now in `altXLabel`, but it's set up to hold multiple chart types.  Would this test always be applicable to everything with any custom x axis label?